index.html

```diff
@@ -138,13 +138,28 @@ <h1>About Myself</h1>
                         <li class="lang-item"><img src="../src/assets/images/Ellipse_css.svg" alt="img">CSS</li>
                     </ul>
                 </li>
-                <li class="lang-items-title underline">Frameworks<span>></span></li>
-                
-                <li class="lang-items-title underline">Skills<span>></span></li>
-                
+                <li class="lang-items-title underline">Frameworks<span>></span></li>                
+                <li class="lang-items-title underline">Skills<span>></span></li>                
             </ul>
         </section>
-        
+        <section class="contact-form">
+            <img class="contact-bg" src="../src/assets/images/Contact_me_bg.svg" alt="image">
+            <img class="shape-position" src="../src/assets/images/contact_form_shapes_1.svg" alt="img">
```

Comment:
- [Optional] Please note that all images that are strictly for decorations should be added as background images in CSS. This is best practice.

- [Optional] Consider fixing all your links from "../ ..." to "./..." since your index.html is in the same folder as the `src` folder and the `css` folder.